### PR TITLE
feat(bedrock): Upgrade contract addresses for sdk

### PR DESCRIPTION
**Pull request by roninjin10**
Land day of bedrock upgrade.  Updates contract addresses post bedrock for sdk

Requires the contract addresses to exist in the deployment artifacts before this pr is viable

### CODE REVIEW

**Comment by maurelian** [comment]
Fixes CLI-4025

File: packages/sdk/src/utils/chain-constants.ts [hunk 1]
```diff
@@ -77,8 +80,8 @@ const getL1ContractsByNetworkName = (network: string): OEL1ContractsLike => {
     StateCommitmentChain: getDeployedAddress('StateCommitmentChain'),
     CanonicalTransactionChain: getDeployedAddress('CanonicalTransactionChain'),
     BondManager: getDeployedAddress('BondManager'),
-    OptimismPortal: '0x5b47E1A08Ea6d985D6649300584e6722Ec4B1383' as const,
-    L2OutputOracle: '0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0' as const,
+    OptimismPortal: getContractDefinition('OptimismPortal').address,
+    L2OutputOracle: getContractDefinition('L2OutputOracle').address,
```
Comment (maurelian) [inline packages/sdk/src/utils/chain-constants.ts:84]: AFAICT `getContractDefinition` only reads the HH artifacts, which don't have an address. 

We might need to port `getDeployedAddress` to the bedrock package? Or you could use [this](https://github.com/ethereum-optimism/optimism/blob/fc2f5887c3e3a14a9c7b2ac11bd0f5bf34895c37/packages/contracts-bedrock/src/deploy-utils.ts#L294), if you're OK with importing the HRE?

Comment (roninjin10) [reply packages/sdk/src/utils/chain-constants.ts]: Absolutely not!!! Do not ever port getDeployedContractDefinition to bedrock.  We need to delete the old one.   It causes major issues with code splitting.

But yea you are right I just replaced it with getting addy directly

**Review by maurelian — APPROVED** [review]
LGTM if CI passes.

File: packages/sdk/src/utils/chain-constants.ts [hunk 2]
```diff
@@ -3,6 +3,20 @@ import {
   getDeployedContractDefinition,
 } from '@eth-optimism/contracts'
 import { predeploys as bedrockPredeploys } from '@eth-optimism/contracts-bedrock'
+import portalArtifactsMainnet from '@eth-optimism/contracts-bedrock/deployments/mainnet/OptimismPortal.json'
+import portalArtifactsGoerli from '@eth-optimism/contracts-bedrock/deployments/goerli/OptimismPortal.json'
+import l2OutputOracleArtifactsMainnet from '@eth-optimism/contracts-bedrock/deployments/mainnet/L2OutputOracle.json'
+import l2OutputOracleArtifactsGoerli from '@eth-optimism/contracts-bedrock/deployments/goerli/L2OutputOracle.json'
```
Comment (maurelian) [inline packages/sdk/src/utils/chain-constants.ts:9]: Pretty sure you need to do this actually:

```suggestion
import portalArtifactsMainnet from '@eth-optimism/contracts-bedrock/deployments/mainnet/OptimismPortalProxy.json'
import portalArtifactsGoerli from '@eth-optimism/contracts-bedrock/deployments/goerli/OptimismPortalProxy.json'
import l2OutputOracleArtifactsMainnet from '@eth-optimism/contracts-bedrock/deployments/mainnet/L2OutputOracleProxy.json'
import l2OutputOracleArtifactsGoerli from '@eth-optimism/contracts-bedrock/deployments/goerli/L2OutputOracleProxy.json'
```

Comment (roninjin10) [reply packages/sdk/src/utils/chain-constants.ts]: Yup fixed test 
image

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**